spec.bs

```diff
@@ -395,14 +395,12 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
 
  1. Let |tokens| be the result of [=generate masked tokens|generating masked tokens=] with |numTokens|.
  1. Configure the HTTP request.
-               a. Set a load flag to bypass the HTTP cache.
-          b. Add <a http-header>Sec-Private-State-Token</a> request header containing a
-                      base64-encoded version of the bytestring of tokens
+               1. Set a load flag to bypass the HTTP cache.
```

Comment:
We should specify this more precisely, but this can happen in a follow up (not exactly sure what kind of flag that is, but I can look it up)